examples/nextjs/package.json

```diff
@@ -9,12 +9,12 @@
         "lint": "next lint"
     },
     "dependencies": {
-        "react": "^19.0.0",
```

Comment:
This makes the versions in the lockfile align better between pnpm 8+9. It seems pnpm v9 changed the ^ to only incremental the last semver specified (which sounds right to me 🤔). Without this some transitive deps change and actually caused this nextjs example to fail.